src/public/style.css

```diff
@@ -51,6 +100,7 @@ button:hover {
   width: 35%;
   height: 25%;
   box-shadow: 5px 5px 2px 0 rgba(0,0,0,0.6);
```

Comment:
it looks you have some css duplication, i guess you could use selector to limit this.